.gitignore

```diff
@@ -0,0 +1,2 @@
+ept/lidar_raleigh_nc_spm_height_feet.las
```

Comment:
Why is this in "gitignore"?

Reply:
because it is about 70MB and are instructions on how to download this raw data with `curl`.